### DISCUSSION TITLE
我怎样才能翻译表格

### DISCUSSION THREAD

**Discussion by farawayfortemple** [immersive-translate/immersive-translate#463] [Help] [answered]
### 概述 | Summary

在 https://2e.aonprd.com/Creatures.aspx 中，我尝试翻译生物表格，但是不论是表格内容还是”筛选器“，“选项”等都无法翻译。

我在单独使用GOOGLE翻译时却可以正常翻译。

这是为什么？我怎样才能使用沉浸式翻译，翻译表格中的内容？

使用版本：V0.5.5
翻译器：GOOGLE

**Comment by chzzfly** [comment]
你可以使用最新的功能“当鼠标滑过”这个功能，我测试了下，还可以。

**Comment by versun** [comment, marked as answer]
我这边可以的哈，”筛选器“，“选项”需要切换为“翻译所有区域”就可以了
![image](https://github.com/immersive-translate/immersive-translate/assets/2398708/ca65cc20-19b1-49d5-967d-407ecc49512e)
有些专有名词无法翻译哈
![image](https://github.com/immersive-translate/immersive-translate/assets/2398708/d80f96a1-dbf5-45af-ac66-e37091ad7958)
